### ISSUE TITLE
using "long" icon with open menu, and small "standard" when menu is compressed 

### ISSUE THREAD

**Issue by anubisg1** [yaminncco/vue-sidebar-menu#235]
What i would like to achieve is that the very first menu entry, would be only an icon taking a wider space so that i can include my complete company logo. 

when the menu is minimized then i would replace that icon with the "favico" icon that's square a fits everything else ...
i'm having some issue doing this.
Of course i'd need to disable hovering too.
i tried

```
        {
          icon: {
            element: "img",
            attributes: {
              src: "/src/assets/company-logo-small-wb.png",
            },
            class: "company-logo"
          },
          disableHover: true,
        },
```

but disabe Hover doesn't seem to work

basically similar to this:

![image](https://user-images.githubusercontent.com/4518931/174612517-78ef6928-2774-4f85-803b-bf6856fec4fb.png)
![image](https://user-images.githubusercontent.com/4518931/174612571-33e79893-53b5-44f1-8496-7114ce3a6bef.png)

any tip , is that even possible? 
thanks a lot for your helo

**Comment by yaminncco** [comment]
I suggest to use header slot

```
<sidebar-menu v-model:collapsed="collapsed">
  <template #header>
    <div v-if="collapsed">
      Logo
    </div>
    <div v-else>
      Long logo
    </div>
  </template>
</sidebar-menu>
```

**Comment by anubisg1** [comment]
this worked well, thank you
